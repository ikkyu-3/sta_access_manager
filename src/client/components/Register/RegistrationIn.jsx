// @flow
import * as React from "react";
import styles from "./style.css";

function RegistrationIn(): React.Node {
  const { main, content, registrationIn } = styles;
  return (
    <div className={main}>
      <div className={`${content} ${registrationIn}`}>登録中</div>
    </div>
  );
}
export default RegistrationIn;
