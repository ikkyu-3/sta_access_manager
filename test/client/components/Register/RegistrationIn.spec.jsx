import React from "react";
import renderer from "react-test-renderer";
import RegistrationIn from "@components/Register/RegistrationIn";

describe("RegistrationIn.jsxのテスト", () => {
  describe("スナッップショット", () => {
    it("正しいレンダリング", () => {
      const tree = renderer.create(<RegistrationIn />);
      expect(tree).toMatchSnapshot();
    });
  });
});
