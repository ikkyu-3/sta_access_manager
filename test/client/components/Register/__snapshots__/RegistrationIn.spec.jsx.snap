// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationIn.jsxのテスト スナッップショット 正しいレンダリング 1`] = `
<div
  className="main"
>
  <div
    className="content registrationIn"
  >
    登録中
  </div>
</div>
`;
